### PR TITLE
Change error type for unhandled exceptions

### PR DESCRIPTION
**Pull request by BartoszLitwiniuk**

### CODE REVIEW

File: backtrace-library/src/androidTest/java/backtraceio/library/SettingAttributesTest.java [hunk 1]
```diff
@@ -177,6 +178,7 @@ public BacktraceResult onRequest(BacktraceData data) {
                         waiter.assertTrue(data.report.attributes.containsKey(customClientAttributeKey));
                         waiter.assertEquals(customClientAttributeValue, data.report.attributes.get(customClientAttributeKey));
                         waiter.assertEquals(exceptionMessage, data.report.exception.getMessage());
+                        waiter.assertEquals(data.report.attributes.get(BacktraceAttributeConsts.ErrorType), BacktraceAttributeConsts.HandledExceptionAttributeType);
```
[the remark this answers is not in the document]
Comment (BartoszLitwiniuk) [reply backtrace-library/src/androidTest/java/backtraceio/library/SettingAttributesTest.java]: We are able to test only `UnhandledExceptionAttributeType` without any advanced mocking

File: backtrace-library/src/main/java/backtraceio/library/models/BacktraceExceptionHandler.java [hunk 2]
```diff
@@ -54,7 +56,7 @@ public void uncaughtException(final Thread thread, final Throwable throwable) {
         if (throwable instanceof Exception) {
             BacktraceLogger.e(LOG_TAG, "Sending uncaught exception to Backtrace API", throwable);
             BacktraceReport report = new BacktraceReport((Exception) throwable, BacktraceExceptionHandler.customAttributes);
-            report.attributes.put(BacktraceAttributeConsts.ErrorType, BacktraceAttributeConsts.UnhandledExceptionAttributeType);
+            report.attributes.put(BacktraceAttributeConsts.ErrorType, isMainThread() ? BacktraceAttributeConsts.CrashAttributeType : BacktraceAttributeConsts.HandledExceptionAttributeType);
```
[the remark this answers is not in the document]
Comment (BartoszLitwiniuk) [reply backtrace-library/src/main/java/backtraceio/library/models/BacktraceExceptionHandler.java]: ok, done

File: backtrace-library/src/main/java/backtraceio/library/models/BacktraceExceptionHandler.java [hunk 3]
```diff
@@ -66,6 +68,10 @@ public void uncaughtException(final Thread thread, final Throwable throwable) {
         }
     }
 
+    private boolean isMainThread() {
+        return Looper.myLooper() == Looper.getMainLooper();
```
[the remark this answers is not in the document]
Comment (BartoszLitwiniuk) [reply backtrace-library/src/main/java/backtraceio/library/models/BacktraceExceptionHandler.java:72]: this methods shouldn't throw any exceptions so I think it is not necessary

**Comment by vlussenburg** [comment]
LGTM!

**Review by vlussenburg — APPROVED** [review]

**Pull request merged**